source/release-notes/backend.txt

```diff
@@ -13,6 +13,35 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20231213:
+
+13 December 2023 Release
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added support for passing in user's new metadata from a request using JWT authentication to the function context.
+
+- Apps, users, and refresh token are now pruned at the 30 day mark.
+
+- Fixed webpack production yarn build warnings, such as "asset size limit" and "entrypoint size warnings", by reducing bundle size. 
+
+- Updated App Services schema UI to display the difference between a 64-bit ``long`` and 32-bit ``integer``.
+
+- Disallowed creating schemas with invalid database or collection names, such as names including spaces.
+
+- Left default for ``allowDiskUse`` as true unless user specifies it as false. 
+
+- Fixed bug where a read document filter set to false is not unioned with a write filter.
+
+- Created a private endpoint for enabling tiered sync.
```

Comment:
```suggestion
- Device Sync permissions now properly union a read filter set to ``false`` with its corresponding write filter.
```

1. For this one I think it's clearer to describe the new behavior than saying what it doesn't not do anymore

2. This is a private endpoint so best to leave it out of the docs